packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -530,7 +681,14 @@ class TimeInput extends React.Component {
     if (event.keyCode === KeyCode.KEY_LEFT
         || event.keyCode === KeyCode.KEY_DELETE
         || event.keyCode === KeyCode.KEY_BACK_SPACE) {
-      this.minuteInput.focus();
+      if (this.props.showSecond) {
+        this.secondInput.focus();
+        if (this.state.second) {
+          this.secondInput.setSelectionRange(this.state.second.length, this.state.second.length);
+        }
+      } else {
+        this.minuteInput.focus();
```

Comment:
Not sure about this statement. This is how it was before, but it is somewhat inconsistent with other behavior. When keyboard navigating from the minute input to hour the cursor will be placed at the right of the input with nothing selected. When keyboard navigating from the meridiem directly to the minute the text inside the minute will be selected. The above represents how other keyboard navigation to text inputs is processed.

Reply:
I think the minute input here should also use setSelectRange to be consistent.

Reply:
This was resolved in b18195c6cfa9959e7f7390c5aed3029b40147f5d